lib/cli/package.json

```diff
@@ -52,8 +53,8 @@
     "fs-extra": "^9.0.0",
     "get-port": "^5.1.1",
     "inquirer": "^7.0.0",
+    "json5": "^2.1.3",
```

Comment:
Order was right before, final reference of this in Meteor generator. Worth removing it !